### DISCUSSION TITLE
upgrading from tvial/docker-mailserver

### DISCUSSION THREAD

**Discussion by szelga** [docker-mailserver/docker-mailserver#2894] [General]
hello.
I have a minimalistic (and semi-neglected) setup of [`tvial/docker-mailserver`](https://hub.docker.com/r/tvial/docker-mailserver) on my hands. would it be too painful to upgrade to `docker-mailserver` using the same data volumes? has anyone done that?

**Comment by casperklein** [comment]
That shouldn't be a problem. Make sure, to read the [CHANGELOG.md](https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md) carefully, to see if any ENV you might be using have changed.

In any case, **make a backup before you upgrade**. So you can easily switch back, if something doesn't work as you expect.

**Comment by s0ftcorn** [comment]
I had a similar case, i compared the current .env to the old one as well as the compose files, made the according changes and all went well. of course your experience may be a different one. A backup is really really important.

**Comment by szelga** [comment]
thanks, gonna try that.

**Comment by mrniko** [comment]
Are the steps below correct if I want to upgrade to 8.0.1 version in docker-compose?

change 
`image: tvial/docker-mailserver:latest`
to 
`image: mailserver/docker-mailserver:8.0.1`

then

`docker-compose down`
`docker-compose up -d mail`

is it possible to add `maillogs` binding in `volumes` if it's missed?

**Reply by casperklein** [reply]
Yes, that should work. Make sure to check the CHANGELOG file and the latest [`compose.yaml`](https://github.com/docker-mailserver/docker-mailserver/blob/master/compose.yaml) (`docker-compose.yaml` in the past) file.
